hadoop-ozone/dist/src/main/smoketest/test.sh

```diff
@@ -24,6 +23,41 @@ mkdir -p "$DIR/$RESULT_DIR"
 #Should be writeable from the docker containers where user is different.
 chmod ogu+w "$DIR/$RESULT_DIR"
 
+## @description wait until 3 datanodes are up (or 30 seconds)
+## @param the docker-compose file
+wait_for_datanodes(){
+
+  #Reset the timer
+  SECONDS=0
+
+  #Don't give it up until 30 seconds
```

Comment:
I think it's fine. the value of sleep is independent as we check the elapsed time based on the $SLEEP environment variable. It will iterate at every 2 seconds until 30 seconds  (If I didn't miss something)

Reply:
Yes, but now I got a question, where is this value SLEEP is incremented, as we set value SECONDS=0, and after that loop, I don't see it is getting modified.